Could not find artifact org.elasticsearch.plugin:ingest-attachment:zip:6.2.1

I'm getting this build failure:

```
[...]
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/shared/maven-invoker/2.1.1/maven-invoker-2.1.1.jar (30 kB at 29 kB/s)
Downloaded from central: https://repo.maven.apache.org/maven2/commons-collections/commons-collections/3.2.1/commons-collections-3.2.1.jar (575 kB at 489 kB/s)
[INFO] Configured Artifact: org.elasticsearch.plugin:ingest-attachment:6.2.1:zip
Downloading from elastic-download-service: https://artifacts.elastic.co/maven/org/elasticsearch/plugin/ingest-attachment/6.2.1/ingest-attachment-6.2.1.zip
Downloading from central: https://repo.maven.apache.org/maven2/org/elasticsearch/plugin/ingest-attachment/6.2.1/ingest-attachment-6.2.1.zip
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 46.642 s
[INFO] Finished at: 2018-05-16T07:38:15+02:00
[INFO] Final Memory: 33M/505M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-dependency-plugin:2.10:copy (integ-setup-dependencies-plugins) on project testcontainers-elasticsearch: Unable to find artifact.: Could not find artifact org.elasticsearch.plugin:ingest-attachment:zip:6.2.1 in elastic-download-service (https://artifacts.elastic.co/maven/)
[ERROR] 
[ERROR] Try downloading the file manually from the project website.
[ERROR] 
[ERROR] Then, install it using the command: 
[ERROR]     mvn install:install-file -DgroupId=org.elasticsearch.plugin -DartifactId=ingest-attachment -Dversion=6.2.1 -Dpackaging=zip -Dfile=/path/to/file
```

--- a/src/main/java/fr/pilato/elasticsearch/containers/ElasticsearchContainer.java
+++ b/src/main/java/fr/pilato/elasticsearch/containers/ElasticsearchContainer.java
@@ -24,6 +24,7 @@ import org.jetbrains.annotations.NotNull;
 import org.testcontainers.containers.BindMode;
 import org.testcontainers.containers.GenericContainer;
 import org.testcontainers.images.builder.ImageFromDockerfile;
+import org.testcontainers.shaded.com.google.common.collect.ImmutableSet;
 
 import java.io.IOException;
 import java.nio.file.Files;
@@ -145,9 +146,7 @@ public class ElasticsearchContainer<SELF extends ElasticsearchContainer<SELF>> e
     @NotNull
     @Override
     protected Set<Integer> getLivenessCheckPorts() {
-        Set<Integer> ports = new HashSet<>(super.getLivenessCheckPorts());
-        ports.add(getMappedPort(ELASTICSEARCH_DEFAULT_PORT));
-        return ports;
+        return ImmutableSet.of(getMappedPort(ELASTICSEARCH_DEFAULT_PORT));
     }
 
     @Override

